custom_metrics/almanac.js

```diff
@@ -277,20 +307,43 @@ return JSON.stringify({
 
     return parsedNodes;
   })(),
-  '12.11': (() => {
-    // Counts the links or buttons only containing an icon.
+  // Counts the links or buttons only containing an icon.
+  'icon_only_clickables': (() => {
+    function containsAnSvg(element) {
+      var children = Array.from(element.childNodes);
+      return !!children.find((child) => {
+        if (child.tagName && child.tagName.toLowerCase() === 'svg') {
+          return true;
+        }
+
+        if (child.childNodes.length) {
+          return containsAnSvg(child);
+        }
+
+        return false;
+      });
+    }
+
     var clickables = document.querySelectorAll('a, button');
     return Array.from(clickables).reduce((n, clickable) => {
-      // Clickables containing SVG are assumed to be icons.
-      if (clickable.firstElementChild && clickable.firstElementChild.tagName == 'SVG') {
-        return n + 1;
-      }
+      var visible_text_length = clickable.textContent.trim().length;
+
       // Clickables containing 1-char text are assumed to be icons.
       // Note that this fails spectacularly for complex unicode points.
       // See https://blog.jonnew.com/posts/poo-dot-length-equals-two.
-      if (clickable.textContent.trim().length == 1) {
+      if (visible_text_length == 1) {
+        return n + 1;
+      }
+
+      if (containsAnSvg(clickable)) {
```

Comment:
Per https://github.com/HTTPArchive/legacy.httparchive.org/pull/167#discussion_r457783825 I think this can be simplified to:
```suggestion
      if (clickable.querySelector('svg')) {
```